app/views/products/edit.html.erb

```diff
@@ -5,6 +5,8 @@
     <p><%= form.text_field :name%></p>
     <p><%= form.label :description %></p>
     <p><%= form.text_area :description, rows: 1  %></p>
+    <p>Категорія</p>
+    <p><%= form.select :category_id, options_for_select(@categories.collect { |cat| [cat.name, cat.id]})%>
```

Comment:
we should be able to add multiple categories

Reply:
no we shouldn't thats useless, we can just add more more precise and diverse categories if needed, also in last commit added filtering so user can find what he wants